pandas/tests/test_algos.py

```diff
@@ -415,6 +415,14 @@ def test_order_of_appearance(self):
         expected = pd.Categorical(list('abc'))
         tm.assert_categorical_equal(result, expected)
 
+    @pytest.mark.parametrize("arg ,expected", [
+        (('1', '1', '2'), np.array(['1', '2'], dtype=object)),
+        (('foo',), np.array(['foo'], dtype=object))
+    ])
+    def test_tuple_with_strings(self, arg, expected):
```

Comment:
Put the issue number (in this case, the PR number) below this function definition (e.g. "see #17108")